src/card.tsx

```diff
@@ -393,3 +406,252 @@ export const SnapCard = React.forwardRef<HTMLDivElement, SnapCardProps>(
         );
     }
 );
+
+type CardAction = {
+    label: string;
+    onPress: () => void;
+    Icon: React.FC<IconProps>;
+};
+
+type CardActionsGroupProps = {
+    actions: Array<CardAction>;
+    isInverse?: boolean;
+};
+
+const CardActionsGroup = ({actions, isInverse}: CardActionsGroupProps): JSX.Element => {
+    return (
+        <Inline space={0}>
+            {actions.map(({onPress, label, Icon}, index) => (
+                <IconButton
+                    size={48}
+                    key={index}
+                    onPress={onPress}
+                    aria-label={label}
+                    className={styles.cardActionIconButton}
+                    style={{display: 'flex'}}
```

Comment:
couldn't this flex be move to `cardActionIconButton`?

Reply:
hmmm... maybe, I'll check it

Reply:
hmmm... no, because there is already a class in IconButton adding `display: inline-block` :(